### PR TITLE
Feature/DAO switcher & abstain vote option 

### PR DESCRIPTION
**Pull request by RakeshUP**
DAO switcher first pass(DAO-715) & abstain voting option added(DAO-745)

The DAO switcher is very basic and is missing out on functionality. We can work on it further after we have some real data and also decide the parameters in urls

### CODE REVIEW

File: packages/ui-components/src/components/table/votersTable.tsx [hunk 1]
```diff
@@ -38,7 +38,13 @@ export const VotersTable: React.FC<VotersTableProps> = ({
             <TableCell type="tag">
               <Badge
                 label={voter.option}
-                colorScheme={voter.option === 'Yes' ? 'success' : 'critical'}
+                colorScheme={
```
Comment (Fabricevladimir) [inline packages/ui-components/src/components/table/votersTable.tsx]: Might want to extract this into a variable.

File: packages/web-app/src/containers/navbar/daoSelectMenu.tsx [hunk 2]
```diff
@@ -7,26 +7,22 @@ import {
...
 
 // NOTE: the state setting is temporary until backend integration
 const DaoSelectMenu: React.FC = () => {
   const {t} = useTranslation();
-  const [selectedDao, setSelectedDao] = useState(DAOS[0]);
+  const selectedDao = useReactiveVar(selectedDAO);
+  const favoriteDaos = useReactiveVar(favoriteDAOs);
   const {isSelectDaoOpen, close} = useGlobalModalContext();
 
   const handleDaoSelect = (dao: {daoName: string; daoAddress: string}) => {
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/navbar/daoSelectMenu.tsx:24]: If a dao is only the name and address, how do we know which network a dao belongs to?

**Review by Fabricevladimir — COMMENTED** [review]
Thanks! The Apollo client local storage variable mechanism is very handy

**Review by WissenIstNacht — CHANGES_REQUESTED** [review]
Thanks Rakesh. I think Fabrice's comments already cover everything. PIng me when they're resolved.

File: packages/web-app/src/containers/navbar/daoSelectMenu.tsx (hunk 2 above)
Comment (RakeshUP) [reply packages/web-app/src/containers/navbar/daoSelectMenu.tsx:24]: I am still not sure of the data structure for the favoriteDAOs array. I just copied it from the existing temp data in Navbar DaoSelector. Things like ENS name for DAO might be added too along with network

Comment (Fabricevladimir) [reply packages/web-app/src/containers/navbar/daoSelectMenu.tsx:24]: I guess that's also going to depend on how we figure out which network a DAO is in as well. Fair enough.

**Review by Fabricevladimir — APPROVED** [review]

**Review by WissenIstNacht — APPROVED** [review]

**Pull request merged**